app/views/admin/products_v3/_table.html.haml

```diff
@@ -13,20 +13,21 @@
   = hidden_field_tag :producer_id, @producer_id
   = hidden_field_tag :category_id, @category_id
 
-  %table.products
+  %table.products{ 'data-column-preferences-target': "table" }
     %colgroup
-      %col{ width:"56" }= #  Img (size + padding)
-      %col= # (grow to fill) Name
-      %col{ width:"5%"}
-      %col{ width:"8%"}
-      %col{ width:"8%"}
-      %col{ width:"5%"}
-      %col{ width:"10%"}
-      %col{ width:"15%"}= # Producer
-      %col{ width:"8%"}
-      %col{ width:"8%"}
-      %col{ width:"8%"}
-      %col{ width:"8%"}= # Actions
+      -# The `min-width` property works in Chrome but not Firefox.
+      %col{ 'data-column-preferences-name': :image,      width:"56px" }= # (image size + padding)
```

Comment:
I restored the image column width to display images again.

Reply:
I'm not sure why different browsers behave differently, but this happens. At least they should deal with web pages in a similar manner 😅

Reply:
Thanks, and I really appreciate the code comment!

Reply:
I guess this is the explanation: https://caniuse.com/?search=min-width

> CSS 2.1 leaves the behavior of min-width with [table](https://developer.mozilla.org/docs/Web/HTML/Element/table) undefined. Firefox supports applying min-width to table elements.

(there is the same comment about Opera too).
This implies that it _doesn't_ support min-width on columns or cells.

Back to the drawing board..